package.json

```diff
@@ -18,13 +17,18 @@
   "author": "Richard Rodger (http://richardrodger.com)",
   "license": "MIT",
   "dependencies": {
-    "async": "~0.9.0"
+    "async": "~0.9.0",
+    "babel-eslint": "6.0.4",
+    "eslint-plugin-react": "5.0.1"
   },
   "files": [
     "README.md",
     "LICENSE.txt",
     "cache-test.js"
   ],
   "devDependencies": {
+    "eslint-config-seneca": "1.1.x",
+    "eslint-plugin-standard": "1.3.x",
+    "lab": "6.2.x"
   }
```

Comment:
the linting dependencies need cleanup, the eslint-plugin-react probably not needed